### PR TITLE
GameDB: Cleanup comments

### PR DESCRIPTION
**Pull request by RedDevilus**
Adding punctation, removing duplicate serial and making english language more happy for reading.

### CODE REVIEW

File: bin/GameIndex.yaml [hunk 1]
```diff
@@ -12676,9 +12677,6 @@ SLES-53446:
   name: "Arcade USA"
   region: "PAL-E"
   compat: 5
-SLES-53450:
```
Comment (lightningterror) [inline bin/GameIndex.yaml:12679]: Why was this removed?

Comment (refractionpcsx2) [reply bin/GameIndex.yaml:12679]: Yeah that's going to break the parsing of the file.

Comment (RedDevilus) [reply bin/GameIndex.yaml:12679]: Sces-53450 is the correct serial. Sles-53450 is wrong. http://redump.org/disc/12328/

Comment (lightningterror) [reply bin/GameIndex.yaml:12679]: @Mrlinkwii Can you confirm with Red what the correct serial is?

Comment (Mrlinkwii) [reply bin/GameIndex.yaml:12679]: done

Comment (refractionpcsx2) [reply bin/GameIndex.yaml:12679]: okay, I see the entry is already in there, Do you want to correct the title for the entry that's already in there, Red?

Comment (RedDevilus) [reply bin/GameIndex.yaml:12679]: I'll do a bigger PR later and include that as well.
